src/gui/properties/propertieswidget.cpp

```diff
@@ -194,13 +194,16 @@ void PropertiesWidget::showPiecesDownloaded(bool show)
 
 void PropertiesWidget::setVisibility(bool visible)
```

Comment:
While we are here, do you mind adding `const` to it?

Reply:
Ofc

Reply:
If I were to make `handleWidth` class variable then this can't be const.

Reply:
oh, I meant `const bool visible`